lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+'''
+Unit Tests for the manage.py command rename_user
+'''
+from django.core.management.base import CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from .. import rename_user
+import sys
```

Comment:
Import statements should be separated into groups, per the style guide [1].

There should be a single, empty line between each group.

Within each groups, import statements should be alphabetized.

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines